test/everypolitician_test.rb

```diff
@@ -30,6 +51,21 @@ def test_legislature_find
     end
   end
 
+  def test_find_legislature_is_case_insensitive
+    VCR.use_cassette('countries_json') do
+      legislature = Everypolitician::Legislature.find('UK', 'commons')
+      assert_equal 'House of Commons', legislature.name
+    end
+  end
+
+  def test_finds_legislature_by_several_fields
+    VCR.use_cassette('countries_json') do
+      query = {lastmod: '1469382789', person_count: 475}
```

Comment:
My original objection was to testing searches against `lastmod` and `person_count`. Those currently work, but I'd say that's by accident rather than design, and I'm wary that explicitly testing them means we're essentially increasing the commitment to keeping them working. (Not guaranteeing it, but making it significantly harder to drop support for it.)

On reflection, however, I think that's _also_ true of the whole concept of being able to pass multiple key-value pairs to `find`. As per the discussion yesterday, that's also something that largely works by accident, rather than being officially supported. 

It's definitely worth raising an issue to simplify the code (or to officially document and test it), but that seems to be too big an `en passant` change for _this_ PR.